plugin.go

```diff
@@ -76,7 +78,7 @@ func setPushEventCommand(p *Plugin) {
 	}
 	if p.Config.Values != "" {
 		upgrade = append(upgrade, "--set")
-		upgrade = append(upgrade, p.Config.Values)
+		upgrade = append(upgrade, unQuote(p.Config.Values))
```

Comment:
I had errors with quotes while testing and had to unquote, not sure why nobody else had these issues but I kept it in.